### DISCUSSION TITLE
How to turn off automatic steps and substeps creation for Cypress?

### DISCUSSION THREAD

**Discussion by tomaszwojciechowski** [allure-framework/allure2#2712] [Documentation] [answered]
Even, when I use coded steps, automatic steps and substeps creation is still turned on.

**Comment by delatrie** [comment, marked as answer]
Hi, @tomaszwojciechowski !
There is no such option currently. If you need one, you may create a feature request in the [allure-js](https://github.com/orgs/allure-framework/allure-js) repository.

**Reply by tomaszwojciechowski** [reply]
Done: https://github.com/allure-framework/allure-js/issues/1129
Thank you
